asv_bench/asv.conf.json

```diff
@@ -126,6 +126,6 @@
     },
     "build_command":
     ["python -m pip install versioneer[toml]",
-     "python setup.py build -j4",
+     "python setup.py build -j1",
```

Comment:
Looks like this is failing asvs?

Reply:
I've reverted this. The asv job was failing before I tried this change. This was just an attempt to find the issue.